README.md

```diff
@@ -6,6 +6,25 @@ Set `GOMAXPROCS` to match container CPU quota.
 
 `go get -u go.uber.org/automaxprocs`
 
+## Quick Start
+
+```go
+import (
+  "log"
+
+  "go.uber.org/automaxprocs"
```

Comment:
I'm fine with the global state mutation, but I really hate the error handling - I'd prefer an API that forces the caller to decide whether they think that errors are critical. I don't mind separating the two into separate packages, though.